UnitTestFrameworkPkg/ReadMe.md

```diff
@@ -411,7 +415,7 @@ Host-Based Mocks and Stubs  | Mock and Stub libraries should live in the `UefiTe
 
 ### If still in doubt...
 
-Hop on GitHub and ask @corthon, @mdkinney, or @spbrogan. ;)
+Hop on GitHub and ask @mdkinney or @spbrogan.

```

Comment:
Please drop individuals and ask them to start a GitHub discussion https://github.com/tianocore/edk2/discussions.

```suggestion
Ask a question in [edk2 discussions](https://github.com/tianocore/edk2/discussions) or reach out on the [EDK II development mailing list \<`devel@edk2.groups.io`\>](mailto:devel@edk2.groups.io).
```